lib/Rozier/src/AjaxControllers/AjaxNodesController.php

```diff
@@ -143,7 +145,7 @@ protected function updatePosition(array $parameters, Node $node): void
         $position = $this->parsePosition($parameters, $node->getPosition());
 
         try {
-            if ($node->getNodeType()->isReachable()) {
+            if ($this->nodeTypesBag->get($node->getNodeTypeName())->isReachable()) {
```

Comment:
Use nullish operator, bags could return `null`

```suggestion
            if ($this->nodeTypesBag->get($node->getNodeTypeName())?->isReachable()) {
```